README.md

```diff
@@ -1,113 +1,348 @@
-# Navferty's Excel Add-In
+##### Документация написана на английском языке. На русском доступна по [ссылке](README_RU.md).
```

Comment:
I would suggest to have a list of all localized versions, starting with small country flag, on top of each version, with hyperlinks (except hyperlink to current version)